.github/workflows/deploy-docs.yml

```diff
@@ -0,0 +1,54 @@
+name: Build and push CLI doc to S3
+on:
+  release:
+    types:
+      - published # Triggered only when a new release is published
+    branches:
+      - master
```

Comment:
The expected behavior is: build and deploy when a new release is published. Could you check if this is the proper configuration?

Reply:
yes we release on master